.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,9 +9,14 @@ http://docs.buildbot.net/latest/developer/quickstart.html
 And especially:
 http://docs.buildbot.net/latest/developer/pull-request.html
 
-
 ## Contributor Checklist:
 
+*Important*: please use infinitive when describing changes- not "Added" but "Add": you are submitting a request to change, you have not change anything yet (in your branch, yes, in the target branch, no).
```

Comment:
I think it's best to move this to a separate section above that is to be removed. Usually most PRs leave the checkboxes in the PR description.

Reply:
would something like "## Commit Message Guidelines" make sense?

Reply:
We already have a documentation for pull-request, with lot of important guidelines in it.

http://docs.buildbot.net/latest/developer/pull-request.html

I'm not saying that this particular rule is less important than the others, but it should imho be managed as part of the others.

Reply:
i overlooked that one :(  there's no point of repeating the same rules in different places...